src/ProfileDefinition.php

```diff
@@ -1,301 +1,305 @@
-<?php
+<?php declare(strict_types=1);
 namespace gossi\code\profiles;
 
-use Symfony\Component\Config\Definition\ConfigurationInterface;
-use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\NodeDefinition;
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
 
 class ProfileDefinition implements ConfigurationInterface {
-
-	public function getConfigTreeBuilder() {
-		$treeBuilder = new TreeBuilder();
-		$rootNode = $treeBuilder->root('formatter');
-
-		$this->addIndentationSection($rootNode);
-		$this->addBracesSection($rootNode);
-		$this->addWhitespaceSection($rootNode);
-		$this->addNewlinesSection($rootNode);
-		$this->addBlanksSection($rootNode);
+	public function getConfigTreeBuilder(): TreeBuilder {
+		$treeBuilder = new TreeBuilder('formatter');
+		/** @var NodeDefinition $node */
```

Comment:
I can see no variable `$node` in the current scope? Or is github diff view tricking me? What's this for?

Reply:
Sorry, this old comment is here for a mistake. Fixed.